go/logictest/runner.go

```diff
@@ -127,8 +133,12 @@ func generateTestFile(harness Harness, f string) {
 		}
 	}()
 
+	ll := newLoggingLock()
```

Comment:
I wouldn't worry about doing any locking in this method (the log output is irrelevant, it's writing stuff to files).